src/fcntl.rs

```diff
@@ -164,8 +164,9 @@ libc_bitflags!(
         /// Similar to `O_DSYNC` but applies to `read`s instead.
         #[cfg(any(target_os = "linux", netbsdlike))]
         O_RSYNC;
-        /// Skip search permission checks.
-        #[cfg(target_os = "netbsd")]
+        /// Open directory for search only. Skip search permission checks on
+        /// later `openat()` calls using the obtained file descriptor.
+        #[cfg(any(target_os = "netbsd", freebsd, solarish, target_os = "fuchsia"))]
```

Comment:
[We don use alias for only 1 platform](https://github.com/nix-rust/nix/blob/master/CONVENTIONS.md#cfg-gates):

```suggestion
        #[cfg(any(target_os = "netbsd", target_os = "freebsd", solarish, target_os = "fuchsia"))]
```